test/store-test.ts

```diff
@@ -0,0 +1,217 @@
+import { jest as Jest } from "@jest/globals";
+import { Options } from "express-rate-limit";
+
+import RedisStore from "../src/index.js";
+
+interface Hit {
+  count: number;
+  expiry: number;
+  timeout?: NodeJS.Timeout;
+}
+
+class MockRedisClient {
+  hits: Map<string, Hit>;
+
+  constructor() {
+    this.hits = new Map<string, Hit>();
+  }
+
+  sendCommand(...args: string[]): number {
+    if (args[0] === "INCR") {
```

Comment:
This must be why the tests are failing - the current mock handles the INCR and PTTL calls, instead of handling a script call

Reply:
We could change this to say if `args[0] === 'EVALSHA'`, increment the key's value and return the pttl for the key instead of parsing the lua script and all.

With this approach though, we will need to change this everytime we change to script.

Reply:
Another option is to use a redis mocking library that supports `EVAL` and scripts. Let me see if I can find something that does that.

Reply:
Agreed.

Reply:
I found https://github.com/stipsan/ioredis-mock - it supports EVALSHA too. I'll change the tests to use this instead 🚀

Reply:
Done in c71b74b 🎉